test/distributed/fsdp/test_checkpoint_wrapper.py

```diff
@@ -100,6 +101,33 @@ def forward(self, *, a=None, b=None):
         out = model(a=inp, b=inp)
         self.assertEqual(2, len(out))
 
+    def test_checkpoint_wrapper_args_kwargs(self):
+        """
+        Tests that checkpoint_wrapper can pass down args / kwargs to configure
+        torch.utils.checkpoint.
+        """
+
+        count = 0
+
+        @contextlib.contextmanager
+        def ctx_manager():
+            nonlocal count
+            count += 1
+            yield
+
+        def get_ctx_mgrs():
+            return (ctx_manager(), ctx_manager())
+
+        torch_utils_checkpoint = torch.utils.checkpoint.checkpoint
+        m = checkpoint_wrapper(
+            torch.nn.Linear(1, 1),
+            checkpoint_fn=torch_utils_checkpoint,
+            use_reentrant=False,
+            context_fn=get_ctx_mgrs,
```

Comment:
I am guessing this is the kwarg. Should we test arg as well?

Reply:
Decided to eliminate `checkpoint_fn_args` support, as there's really no practical way to use this.